mybookstore/src/index.css

```diff
@@ -1,45 +1,54 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
 :root {
   --primary: #eb7d34;
   --secondary: #ada7a6;
   --dark: #1c1b1b;
   --light: #eee;
 }
 
-* {
-  margin: 0;
-  padding: 0;
-  font-family: 'Arial', sans-serif;
-}
-
 html,
 body {
   display: grid;
   height: 100%;
-  text-align: center;
   place-items: center;
   background: #dde6f0;
 }
 
 .main-heading {
-  color: rgb(16, 120, 216);
+  width: 15rem;
+  height: 2.313rem;
+  font-family: Montserrat;
```

Comment:
- [OPTIONAL] I noticed you are using this font family in other sections. How about making it a single CSS class and applying it to tags that will need such a font family? I think it will help deal with repetition.

See example:
```
.main-font {
   font-family: Montserrat;
}

// in a component's tag
<h1 class="main-font">Title</h1>
```